JavaScript/MergeSort/Mergesort.js

```diff
@@ -0,0 +1,33 @@
+function merge_sort(left_part,right_part) 
```

Comment:
Hi, @adithyanarayana almost there. Just a last few changes.
`merge_sort` --> `merge`, as it is called by this name in mergeSort funcion.